From e13d4688aceb822c5afcfee9910d497729e5f1dd Mon Sep 17 00:00:00 2001
From: Li Wang <li.wang@windriver.com>
Date: Mon, 8 Jul 2013 11:13:11 +0800
Subject: [PATCH 4/7] trace: compile warning: variable may be used
 uninitialized as return value

warning: 'xxx' may be used uninitialized in this function [-Wuninitialized]

Signed-off-by: Li Wang <li.wang@windriver.com>
---
 kernel/trace/trace_events_filter.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/trace/trace_events_filter.c b/kernel/trace/trace_events_filter.c
index 289197a..4d7cd25 100644
--- a/kernel/trace/trace_events_filter.c
+++ b/kernel/trace/trace_events_filter.c
@@ -2006,7 +2006,7 @@ static int ftrace_function_set_regexp(struct ftrace_ops *ops, int filter,
 static int __ftrace_function_set_filter(int filter, char *buf, int len,
 					struct function_filter_data *data)
 {
-	int i, re_cnt, ret;
+	int i, re_cnt, ret = 0;
 	int *reset;
 	char **re;
 
-- 
1.7.10.4

